tests/Admin/AdminTest.php

```diff
@@ -2243,6 +2243,13 @@ public function testDefaultDashboardActionsArePresent(string $objFqn, string $ex
         $this->assertArrayHasKey('create', $admin->getDashboardActions());
     }
 
+    /**
+     * NEXT_MAJOR: Remove the assertion about isDefaultFilter method.
+     *
+     * @legacy
+     *
+     * @expectDeprecation Method "Sonata\AdminBundle\Admin\AbstractAdmin::isDefaultFilter" is deprecated since sonata-project/admin-bundle 3.x.
```

Comment:
My bad
```suggestion
     * @expectedDeprecation Method "Sonata\AdminBundle\Admin\AbstractAdmin::isDefaultFilter" is deprecated since sonata-project/admin-bundle 3.x.
```